README.rst

```diff
@@ -56,13 +56,12 @@ Usage Example
 
 .. code-block:: python
 
-	from busio import I2C
 	import adafruit_bme680
 	import time
 	import board
 
-	# Create library object using our Bus I2C port
-	i2c = I2C(board.SCL, board.SDA)
+	# Create sensor object, communicating over the board's default I2C bus
+    i2c = board.I2C()   # uses board.SCL and board.SDA
```

Comment:
needs another tab?

Reply:
You are right, thanks...

Reply:
it renders ok in my pc, strange
![image](https://user-images.githubusercontent.com/34255413/115975446-ca8b3280-a532-11eb-82a1-bf56f242e9f8.png)